### PR TITLE
Greek description

### PR DESCRIPTION
**Pull request by RaptaG**
Still WIP

### CODE REVIEW

File: Description/el-GR.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Ελληνικά (Ελλάδας)
+
+Ένα πακέτο τροποποιήσεων βασισμένο στο Fabric, το οποίο στοχεύει σε βελτιώσεις απόδοσης και γραφικών.
+
+#### Αναθεώριση του πακέτου στα Αγγλικά:
```
Comment (Madis0) [inline Description/el-GR.md:5]:
```suggestion
Αναθεώριση του πακέτου στα Αγγλικά:
```

I don't see a need for that header format here, unless you want to propose a new row for English as well.

Comment (RaptaG) [reply Description/el-GR.md:5]: It just makes it more visible idk if u want I can remove it

Comment (RaptaG) [reply Description/el-GR.md:5]: Or should I bold the text?

Comment (Madis0) [reply Description/el-GR.md:5]: My point is that if there is a highlighted text in one language, it should be in all.

**Comment by Madis0** [comment]
So is it ready now?

**Comment by RaptaG** [comment]
Yep @Madis0

**Pull request merged**